assets/app/styles/_core.less

```diff
@@ -301,6 +301,14 @@ html, body {
   }
 }
 
+label.required:before {
+  content: '* ';
+  // Use position: absolute instead of negative margin so the asterisk shows
+  // up when overflow is hidden.
+  position: absolute;
+  left: 10px;
```

Comment:
You're right, this is wrong. It just happened to work :(

I'm not sure how to fix the problem with `overflow: hidden;`